Improve Mobile UI for dialogs and action dropdowns

allozaur opened the pull request:
Close #16094

- Fixes mobile sidebar conversation items actions interactivity
- Improves Alert Dialog and Dialog mobile UI
- Adds Alert Dialog for `Reset to Default` in Settings Dialog

https://github.com/user-attachments/assets/289f6986-7474-45e7-bba6-f5efe21ebcd7

allozaur committed:
fix: Always show conversation item actions

allozaur committed:
feat: Improve Alert Dialog and Dialog mobile UI

allozaur committed:
feat: Add settings reset to default confirmation

allozaur committed:
fix: Close Edit dialog on save

allozaur committed:
chore: update webui build output

ServeurpersoCom commented:
Not sure I have all commits in my testing branch, let’s double-check if the z-index issue is still present ?

<img width="833" height="607" alt="Sans titre" src="https://github.com/user-attachments/assets/d91c73db-ce15-4b7a-b4e9-52722892b7da" />

allozaur committed:
Merge remote-tracking branch 'ggml-org/master' into 16094-keep-conversation-actions-usable-on-mobile

allozaur committed:
Merge remote-tracking branch 'ggml-org/master' into 16094-keep-conversation-actions-usable-on-mobile

ServeurpersoCom committed:
webui: implement proper z-index system and scroll management

- Add CSS variable for centralized z-index control
- Fix dropdown positioning with Settings dialog conflicts
- Prevent external scroll interference with proper event handling
- Clean up hardcoded z-index values for maintainable architecture

ServeurpersoCom committed:
webui: ensured the settings dialog enforces dynamic viewport height on mobile while retaining existing desktop sizing overrides

ngxson commented:
Feel free to merge when you're ready @allozaur

allozaur committed:
Merge remote-tracking branch 'origin/master' into 16094-keep-conversation-actions-usable-on-mobile

allozaur committed:
Merge remote-tracking branch 'ServeurpersoCom/z-index-scroll' into 16094-keep-conversation-actions-usable-on-mobile

allozaur committed:
feat: Use `dvh` instead of computed px height for dialogs max height on mobile

allozaur committed:
chore: update webui build output

allozaur committed:
feat: Improve Settings fields UI

allozaur committed:
chore: update webui build output

allozaur committed:
Merge remote-tracking branch 'origin/master' into 16094-keep-conversation-actions-usable-on-mobile

allozaur committed:
chore: update webui build output
